website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -111,6 +111,19 @@ and [after](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c
 
 1. (Optionally) upgrade Envoy to the latest version supported in Consul 1.10.0.
 
+### Transparent Proxy on Kubernetes
+
+When upgrading to Consul >= 1.10.0 and Consul-helm >= 0.32.0 and Consul-k8s >= 0.26.0, a change will be necessary for existing services:
+
+- A Kubernetes Service must be added for every service registered to Consul. This Service should be added before
+performing the upgrade. This will allow services to be managed by a central component which will enable features like
```

Comment:
```suggestion
performing the upgrade. This will allow services to be managed by a central component, called `endpoints-controller`, which will enable features like
```